app/controllers/api/v1/recordings_controller.rb

```diff
@@ -15,21 +15,7 @@ def index
 
         recordings = current_user.recordings&.order(sort_config)&.search(params[:search])
 
-        # TODO: Optimise this.
-        recordings.map! do |recording|
-          {
-            id: recording.id,
-            record_id: recording.record_id,
-            name: recording.name,
-            length: recording.length,
-            users: recording.users,
-            visibility: recording.visibility,
-            created_at: recording.created_at.strftime('%A %B %e, %Y %l:%M%P'),
-            formats: recording.formats
-          }
-        end
-
-        render_json data: recordings, status: :ok
+        render_data data: recordings
```

Comment:
I think it may be better to make specifying the Sterilizers in use a pattern to easily jump to them when needed.

Reply:
I think it's actually cleaner to let the Serializer do its job and figure it out itself - in the cases where we don't want to use the standard way, we can manually specify it